src/chrome/Debugger.js

```diff
@@ -0,0 +1,18 @@
+/// <reference path="Types.js" />
+
```

Comment:
I find an overview comment at the top of the file useful. WDYT?

I am actually curious why this file is named as a "Debugger".

Reply:
> I find an overview comment at the top of the file useful. WDYT?

👍 

> I am actually curious why this file is named as a "Debugger".

This file defines types of the chrome extension `chrome.debugger` api that the audion extension uses. I think it could have different casing or different prefixes. Since it is defining the of the api of that name and that the api wraps the Chrome Debugger Protocol, I don't think naming it Inspector is right. I think a file comment will help alot yeah.

Reply:
I title cased it since it is defining the `Debugger` api type, like other title cased files.